### DISCUSSION TITLE
Node Synced but Wallet not synced

### DISCUSSION THREAD

**Discussion by shadakom** [Chia-Network/chia-blockchain#5023] [Support]
Hi, is this normal?, my node is synced (after so much trouble), but my wallet says not synced and my plots are farming, did i miss something? by the way, version 1.1.5

![Node synced](https://user-images.githubusercontent.com/84208004/118337650-ddfa3f80-b4d9-11eb-9578-3046f696587f.jpg)
![wallet not synced](https://user-images.githubusercontent.com/84208004/118337659-e18dc680-b4d9-11eb-9f99-6e78937a4f23.jpg)
![farming](https://user-images.githubusercontent.com/84208004/118337666-e3f02080-b4d9-11eb-86fb-df33687014bf.jpg)

Thanks in advance for your help.

**Comment by SackofFluor** [comment]
from what I see a lot of people are having that problem (including me).I have, no idea how to fix it, no prober work around.... so its most likely  not a problem for the higher echolon...

**Reply by shadakom** [reply]
done.
i managed to restart the syncing on the wallet...
i stopped chia process with powershell "get-process chia* | stop-process"
i stopped all chia process running in task manager 
i moved the files on the C:\Users\myusername\ .chia\mainnet\wallet\db and let it empty
start chia again and it started to sync from 0
i realize that the file needed was "blockchain_wallet_v1_mainnet_"wallet_number_here".sqlite" this is the file that start growing as it syncs
i stopped all process again and moved back that 1 file... it displayed the status syncing but nothing else happened, there was not movement on the numbers...it tooks some minutes and the numbers started to move. 

![wallet](https://user-images.githubusercontent.com/84208004/118345840-e6b03d00-b4fc-11eb-816e-27f6e00331b6.jpg)

**Comment by cccat6** [comment]
I do have the same problem. Seems wallet sync is always behind full node sync and it will eat up a full thread. Even in this case, it still slow. I guess it might be some problem in the program because my full node is hosted locally, it should be extremely fast to sync.

**Reply by cccat6** [reply]
However, after a long long time of waiting. My wallet finally gets synced. Same height as the full node!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
